starwars/src/Footer.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import lambdaLogo from './lambda-logo.png';
+import redux from './Redux.png';
+
+export default function Footer() {
+  return (
+    <footer>
+       <figure>
+            <div>
+                <p>Built with: </p>
+                <img src={redux} />
+            </div>
+
+            <div>
+                <p>Project of: </p>
+                <a href='http://www.lambdaschool.com' target="_blank"> <img className='lambda' src={lambdaLogo} /></a>
```

Comment:
you remove the console.warning when you have the target _blank attribute, you will want to add a rel="noopener noreferrer" attribute the the a tag also. this is a security thing regarding the opened window still linking back to your site and the possibility of XSS